contrib/completion/zsh/_docker

```diff
@@ -788,7 +788,7 @@ __docker_node_commands() {
         "ls:List nodes in the swarm"
         "promote:Promote a node as manager in the swarm"
         "rm:Remove one or more nodes from the swarm"
-        "ps:List tasks running on a node"
+        "ps:List tasks running on a node, defaults to self"
```

Comment:
"current node" for some definition of "current node".

This is going to list tasks for the node on which the request ingresses the engine-api and would be limited to manager nodes. This is probably okay.